code/datums/components/ranged_attacks.dm

```diff
@@ -66,6 +66,7 @@
 	else
 		target_zone = ran_zone()
 	casing.fire_casing(target, firer, null, null, null, target_zone, 0,  firer)
+	casing.update_desc()
```

Comment:
update_appearance instead? idk if the icon updates

Reply:
Yes you are right, thanks for catching this, I didnt even know they had different sprites.